# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e/no-valgrind.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- e/configure.in_orig	2005-12-24 09:22:32.825991664 +0100
+++ e/configure.in	2005-12-24 15:40:40.962870592 +0100
@@ -39,27 +39,30 @@
 # Setting have_valgrind to "no" seems pointless, but we just need to
 # put something in as the 4th parameter, so configure doesn't abort
 # when valgrind.pc isn't found.
-have_valgrind="no"
-PKG_CHECK_MODULES(VALGRIND, valgrind >= 2.4.0, have_valgrind=yes, have_valgrind=no)
-AC_ARG_ENABLE(valgrind,
-  [  --enable-valgrind               enable valgrind support], [
-      if test x"$enableval" = x"yes" ; then
-        AC_MSG_RESULT(yes)
-        have_valgrind="yes"
-      else
-        AC_MSG_RESULT(no)
-		have_valgrind="no"
-        VALGRIND_CFLAGS=""
-        VALGRIND_LIBS=""
-      fi
-  ], [
-    have_valgrind=$have_valgrind
-  ]
-)
+#have_valgrind="no"
+#PKG_CHECK_MODULES(VALGRIND, valgrind >= 2.4.0, have_valgrind=yes, have_valgrind=no)
+#AC_ARG_ENABLE(valgrind,
+#  [  --enable-valgrind               enable valgrind support], [
+#      if test x"$enableval" = x"yes" ; then
+#        AC_MSG_RESULT(yes)
+#        have_valgrind="yes"
+#      else
+#        AC_MSG_RESULT(no)
+#	have_valgrind="no"
+#        VALGRIND_CFLAGS=""
+#        VALGRIND_LIBS=""
+#      fi
+#  ], [
+#    have_valgrind=$have_valgrind
+#  ]
+#)
+
+#if test x$have_valgrind = "xyes"; then
+#  AC_DEFINE(HAVE_VALGRIND, 1, [Valgrind support])
+#fi
 
-if test x$have_valgrind = "xyes"; then
-  AC_DEFINE(HAVE_VALGRIND, 1, [Valgrind support])
-fi
+AC_SUBST(VALGRIND_CFLAGS)
+AC_SUBST(VALGRIND_LIBS)
 
 MODULE_ARCH="$host_os-$host_cpu"
 AC_SUBST(MODULE_ARCH)
